### PR TITLE
Fix for #16: Add a 404 page

### PR DESCRIPTION
**Pull request by marzepani**
Added 404 route and changed email to team@tessel.io
([Issue #16](https://github.com/tessel/tessel.io/issues/16))

### CODE REVIEW

File: views/error.jade [hunk 1]
```diff
@@ -12,7 +12,10 @@ block content
         h2 Something went wrong with that request :(
     .row(style="margin-bottom:20px;")
       .large-12.small-centered.columns(style="text-align:center")
-        h4 Sorry about that. Need help? Email support@technical.io.
+        h4
+          | Sorry about that. Need help? Email 
+          a(href='mailto:team@tessel.io') team@tessel.io
```
Comment (Frijol) [inline views/error.jade:17]: Good catch!

**Comment by Frijol** [comment]
Hm, I'm still getting 
screen shot 2017-05-23 at 11 28 52 am

when I run locally

**Review by Frijol — CHANGES_REQUESTED** [review]

**Comment by marzepani** [comment]
Had to move the router below the static files check. Please re-test. Thx
screen shot 2017-05-24 at 11 51 28

**Comment by marzepani** [comment]
@Frijol do you still get no error page?

**Comment by Frijol** [comment]
Looks good, thanks!

**Review by Frijol — APPROVED** [review]

**Pull request merged**